projects/nim-lang.org/package.yml

```diff
@@ -0,0 +1,46 @@
+distributable:
+  url: https://nim-lang.org/download/nim-{{ version }}.tar.xz
+  strip-components: 1
+
+# FIXME: test more versions
+versions:
+  - v1.6.14
```

Comment:
is there are URL that we can scrape? /cc @jhheider

Reply:
I think we can use GitHub tags. (Commited)